crates/env/src/engine/on_chain/ext.rs

```diff
@@ -28,6 +28,7 @@ macro_rules! define_error_codes {
     ) => {
         /// Every error that can be returned to a contract when it calls any of the host functions.
         #[repr(u32)]
+        #[allow(clippy::enum_variant_names)]
```

Comment:
Not a fan of having to apply this for the entire set of variants as opposed to just the single problematic one. Given the nature of the macro I don't think there's another way to go though

Reply:
:man_shrugging:

Reply:
@agryaznov do you think we can change the name on the `contracts-pallet` side to make this nicer?

Reply:
I think we can, as the corresponding api function is still unstable. I'll create a PR to pallet. @pmikolajczyk41 sorry it seems that your initial name for the error was a better option indeed 😅

Reply:
changed on the pallet side: https://github.com/paritytech/substrate/pull/13412

Reply:
thanks!